docs/how-to/h-manage-units.md

```diff
@@ -1,62 +1,68 @@
 # How to manage units
 
-Unit management guide for scaling and running admin utility scripts.
+For general Juju unit management process, see the [Juju documentation](https://juju.is/docs/juju/manage-units).
 
-## Replication and scaling
+## Scale the cluster
 
-Increasing the number of Apache Kafka brokers can be achieved by adding more units
-to the Charmed Apache Kafka K8s application:
+Scaling the cluster does not automatically rebalance existing topics and partitions. Rebalancing must be performed manually—before scaling in or after scaling out.
+
+[note type="caution"]
+Reassign partitions **before** scaling in to ensure that decommissioned units do not hold any data. Failing to do so may lead to data loss.
+[/note]
+
+To scale Charmed Apache Kafka application, use `juju scale-application` command with the name of the app and the required number of units:
 
 ```shell
-juju add-unit kafka-k8s -n <num_brokers_to_add>
+juju scale-application kafka-k8s <units>
 ```
 
-For more information on how to manage units, please refer to the [Juju documentation](https://juju.is/docs/juju/manage-units)
+See the `scale-application` [command reference](https://documentation.ubuntu.com/juju/latest/reference/juju-cli/list-of-juju-cli-commands/scale-application/index.html).
 
-It is important to note that when adding more units, the Apache Kafka cluster will not 
-*automatically* rebalance existing topics and partitions. New storage and new brokers
-will be used only when new topics and new partitions are created. 
+Make sure to reassign partitions and topics to use newly added units for existing topics and partitions. See below for guidance.
```

Comment:
``todo:** `Make sure to reassign any existing partitions and topics to the newly added units. See below for guidance`